lib/kennel/syncer.rb

```diff
@@ -182,5 +178,18 @@ def tracking_id(a)
     def tracking_field(a)
       a[:message] ? :message : :description
     end
+
+    def filtered_resources_from_dd
+      Progress.progress "Downloading definitions" do
```

Comment:
I liked the progress outside ... can make filter a method and pass it `download_definitions ` ?

Reply:
Or I can just move the `Progress.progress "Downloading definitions" ...` back to the calling function?